components/SbcSigninButton.vue

```diff
@@ -9,64 +9,24 @@
         v-on="on"
       >
         <span>Log in to my BC Registries Account</span>
-        <v-icon class="mr-n1 ml-2">mdi-menu-down</v-icon>
+        <v-icon class="mr-n1 pl-2">mdi-menu-down</v-icon>
       </v-btn>
     </template>
+    <SbcAuthMenu />
```

Comment:
This common component does a lot more than just redirect. (It updates the store and session storage.)

Also, using this common component, THIS app doesn't have to know the signin routes (bcsc, bceid, idir).